src/navigation/store/NewDeliveryPickup.js

```diff
@@ -0,0 +1,425 @@
+                placeholder={t('STORE_NEW_DELIVERY_ENTER_CONTACT_NAME')}
+              />
+              {errors.contactName && touched.contactName && (
+                <Text note style={styles.errorText}>
+                  {errors.contactName}
+                </Text>
+              )}
+            </View>
+            <View style={[styles.formGroup]}>
+              <Text style={styles.label}>
+                {t('STORE_NEW_DELIVERY_PHONE_NUMBER')}
+              </Text>
+              <FormInput
+                autoCorrect={false}
+                keyboardType="phone-pad"
+                returnKeyType="done"
+                onChangeText={value =>
+                  handleChangeTelephone(value, setFieldValue, setFieldTouched)
+                }
+                onBlur={handleBlur('telephone')}
+                value={values.telephone}
+                placeholder={t('STORE_NEW_DELIVERY_ENTER_PHONE_NUMBER')}
+              />
+              {errors.telephone && touched.telephone && (
+                <Text note style={styles.errorText}>
+                  {errors.telephone}
+                </Text>
+              )}
+            </View>
+            <View style={[styles.formGroup]}>
+              <Text style={styles.label}>
+                {t('STORE_NEW_DELIVERY_ADDRESS_DESCRIPTION')}{' '}
+                <Text style={styles.optional}>({t('OPTIONAL')})</Text>
+              </Text>
+              <FormInput
+                style={{
+                  height: 80,
+                }}
+                autoCorrect={false}
+                multiline={true}
+                onChangeText={handleChange('description')}
+                onBlur={handleBlur('description')}
+                value={values.description}
+                placeholder={t('STORE_NEW_DELIVERY_ENTER_ADDRESS_DESCRIPTION')}
+              />
+            </View>
+          </View>
+        </ModalFormWrapper>
+      )}
+    </Formik>
+  );
+}
+
+const styles = StyleSheet.create({
+  disabled: {
+    display: 'none',
+  },
+  autocompleteWrapper: {
+    height: 40,
+    ...Platform.select({
+      android: {
+        flex: 1,
+        top: 0,
+        right: 0,
+        left: 0,
+        zIndex: 1,
+      },
+      ios: {
+        top: 0,
+        right: 0,
+        left: 0,
+        zIndex: 10,
+        overflow: 'visible',
+      },
+    }),
+  },
+  label: {
+    marginBottom: 8,
+    fontWeight: '600',
+  },
+  optional: {
+    fontWeight: '400',
+    opacity: 0.7,
+    fontSize: 12,
+  },
+  help: {
+    marginBottom: 5,
+    fontWeight: '400',
+  },
+  errorInput: {
+    borderColor: '#FF4136',
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignContent: 'center',
+    gap: 5,
+  },
+  formGroup: {
+    marginBottom: 10,
+  },
+  textInput: {
+    height: 40,
+    paddingHorizontal: 10,
+  },
+  errorText: {
+    color: '#FF4136',
+    marginTop: 5,
+  },
+  textarea: {
+    minHeight: 25 * 3,
+  },
+});
+
+function mapDispatchToProps(state) {
```

Comment:
I'd suggest replacing these with `useSelector` and `useDispatch`

Reply:
const { t } = useTranslation()